demos/using-dev-build/test.ts

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env ts-node
+/* This starts the proxy and the test suite. The proxy configuration is read
+ * from dfx.json and canister_ids.json. The proxy is shutdown after the tests
+ * are run.
+ * This expects the replica to be running, and expects the II canister to have
+ * been deployed.
+ */
+
+import * as fs from "fs";
+import { ChildProcess, spawn } from "child_process";
+
+// Port and URL for II and the sign in app, then exported to the tests
+const II_DAPP_PORT = 8086;
+const II_DAPP_URL = `http://localhost:${II_DAPP_PORT}`;
+
+const WEBAPP_PORT = 8087;
+const WEBAPP_URL = `http://localhost:${WEBAPP_PORT}`;
+
+type Arguments = { noRun: boolean };
+
+/** Parse CLI arguments
+ */
+function parseArguments(): Arguments {
+  if (!(process.argv.length == 2 || process.argv.length == 3)) {
```

Comment:
- `===` not `==` (on line 28 as well)
- fancy suggestion: `!([2, 3].includes(process.argv.length))`